qemu/tests/x86_cpu_model.py

```diff
@@ -6,6 +6,37 @@
 from virttest import error_context, utils_misc, env_process
 
 
+def check_flags(params, flags, test, session=None):
+    """
+    Check cpu flags on host or guest.(only for Linux now)
+    :param params: Dictionary with the test parameters
+    :param flags: checked flags
+    :param test: QEMU test object
+    :param session: guest session
+    """
+    if session:
+        if params["os_type"] == "linux":
+            error_context.context("Check cpu flags inside guest", logging.info)
+            cmd = params["check_flag_cmd"]
+            out = session.cmd_output(cmd).split()
+            missing = [f for f in flags.split() if f not in out]
+            if missing:
+                test.fail("Flag %s not in guest" % missing)
+            no_flags = params.get("no_flags")
+            if no_flags:
+                err_flags = [f for f in no_flags.split() if f in out]
+                if err_flags:
+                    test.fail("Flag %s should not be present in guest" % err_flags)
+    else:
+        check_host_flags = params.get_boolean("check_host_flags")
+        if check_host_flags:
```

Comment:
Suggest to move above two lines out of function.

Reply:
update, please help to review.

Reply:
I meant to move `if check_host_flags:`  to x86_cpu_flags.py before this function.

Reply:
ok, updated.